dashboard/app/models/levels/bubble_choice.rb

```diff
@@ -102,20 +102,31 @@ def summarize(script_level: nil, user_id: nil)
   def summarize_sublevels(script_level: nil, user_id: nil)
     summary = []
     sublevels.each_with_index do |level, index|
-      level_info = {
-        id: level.id,
-        display_name: level.display_name || level.name,
-        description: level.try(:bubble_choice_description),
-        thumbnail_url: level.try(:thumbnail_url),
-        summary: level.summary_for_lesson_plans
-      }
+      level_info = level.summary_for_lesson_plans
+
+      alphabet = ('a'..'z').to_a
+
+      level_info.merge!(
+        {
+          id: level.id,
+          description: level.try(:bubble_choice_description),
+          thumbnail_url: level.try(:thumbnail_url),
+          position: index + 1,
+          letter: alphabet[index],
```

Comment:
Won't the position and letter always be correlated? Do we need both or just position that keys into a hash like { 1 : a, 2 : b ...} when the level is displayed.  I don't feel strongly about this at all, just curious.

Reply:
Ya I debated on finding a different way to do this but I'm worried about curriculum building wanting both for the template and it not being easy to parse inside the template so I went with this for now.